cpp/arcticdb/storage/s3/s3_storage.hpp

```diff
@@ -227,8 +227,13 @@ auto get_s3_config(const ConfigType& conf) {
     client_configuration.maxConnections = conf.max_connections() == 0 ?
             ConfigsMap::instance()->get_int("VersionStore.NumIOThreads", 16) :
             conf.max_connections();
-    client_configuration.connectTimeoutMs = conf.connect_timeout() == 0 ? 30000 : conf.connect_timeout();
-    client_configuration.requestTimeoutMs = conf.request_timeout() == 0 ? 200000 : conf.request_timeout();
+    client_configuration.connectTimeoutMs = ConfigsMap::instance()->get_int(
+            "S3.ConnectionTimeout",
+            conf.connect_timeout() == 0 ? 30000 : conf.connect_timeout());
+    client_configuration.requestTimeoutMs = ConfigsMap::instance()->get_int(
```

Comment:
This only affects Windows? https://sdk.amazonaws.com/cpp/api/LATEST/aws-cpp-sdk-core/html/struct_aws_1_1_client_1_1_client_configuration.html